.gitignore

```diff
@@ -16,7 +16,6 @@ downloads/
 eggs/
 .eggs/
 k6_results
-lib/
```

Comment:
Since `lib` is one of the default folder names created when generating packages/components with Nx I've removed it from the gitignore. This line seems to have come from a template .gitignore, and removing it didn't cause any new files to be discovered by git.